compiler/rustc_builtin_macros/src/source_util.rs

```diff
@@ -216,7 +216,7 @@ pub fn expand_include_bytes(
         }
     };
     match cx.source_map().load_binary_file(&file) {
-        Ok(bytes) => base::MacEager::expr(cx.expr_byte_str(sp, bytes)),
+        Ok(bytes) => base::MacEager::expr(cx.expr_included_bytes(sp, file, bytes)),
```

Comment:
Why not defer loading the file until ast lowering?

Reply:
I considered that, but erroring early if the file couldn't be read seems like a better idea